layouts/default/split_3x6_3/readme.md

```diff
@@ -0,0 +1,3 @@
+# split_3x6_3
+
+   LAYOUT_split_3x6_3
```

Comment:
```suggestion
   LAYOUT_split_3x6_3
   
   Layout macro for 3x6 split keyboards with 3 keys thumb cluster.
```
I think a description would should be provided to help others understand the naming. You can probably come up with something better than me.